### ISSUE TITLE
`VK_KHR_buffer_device_address` extends `VkMemoryAllocateFlagBits` but doesn't require it

### ISSUE THREAD

**Issue by Rua** [KhronosGroup/Vulkan-Docs#1966]
In the spec for the `VK_KHR_buffer_device_address` extension, it says that it extends the `VkMemoryAllocateFlagBits` type with new values. But that type is only available from Vulkan 1.1 or with the `VK_KHR_device_group` extension, and the `VK_KHR_buffer_device_address` extension requires neither of these. This leads to a problem further down the line, because you are required to use the two added bit values for certain VUIDs, but you can't include them if `VkMemoryAllocateFlagBits` is not available.

**Comment by oddhack** [comment]
Agreed about the dependency issue. The current schema does not support this sort of dependency, but we're working on a more general dependency syntax that will allow it. This is taking a while since it's a breaking change that can affect downstream consumers like the validation layers and CTS, but once that's staged we'll be able to update the XML. At a guess that will be sometime in January.
